### DISCUSSION TITLE
Dimensions of part of model instead of whole model

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4530] [Q&A]
Did anyone try or does anyone know how to get the dimensions to show for a specific part of a model instead of the whole model. I use the dimensions from the examples with some visual modifications. The example should explain it a bit better, I am trying to show the dimensions of this model without the height of the cable.

![image_2023_10_24T10_46_14_266Z](https://github.com/google/model-viewer/assets/92090684/8e6ccd01-e5ce-4a85-8dca-9e4624528a5b)
![image_2023_10_24T10_46_44_074Z](https://github.com/google/model-viewer/assets/92090684/2f6ede58-a93d-45bb-ab45-0a32199e934b)

**Comment by elalish** [comment]
There's not a great way to specify what parts you would mean - You'll just have to measure it yourself instead of making it fully automated, or remove that part from the file.

**Reply by danilokrulj** [reply]
I just set them manually for now, maybe one day i will work on a function that will do it automatically. Thanks anyway :)
